dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResource.java

```diff
@@ -501,6 +497,22 @@ private void setUpTimeMachineIfPresent(final PageRenderParams renderParams) {
         }
     }
 
+
+    /**
+     * Determines if the FTM logic should be applied based on the given timeMachineDate.
+     * It checks if the date is older than the grace window (not too recent),
+     * using a configurable time limit.
+     *
+     * @param timeMachineDate The Time Machine date from the request.
+     * @return true if the timeMachineDate is older than the grace window, meaning FTM logic should be applied,
+     *         false otherwise (if within the grace window).
+     */
+    private boolean isOlderThanGraceWindow(final Instant timeMachineDate) {
+        final int graceWindowMinutes = Config.getIntProperty("FTM_GRACE_WINDOW_LIMIT", 5);
```

Comment:
just keep in mind, if the Config is called in a high concurrency context, it could end up blocking the thread
consider using Lazy, if you refresh you can also AtomicReference with Integer and implement EventSubscriber<SystemTableUpdatedKeyEvent> in order to refresh the property dynamically